new addon purchase_requisition_auto_rfq_bid_selection

gurneyalex opened the pull request:
bridge module between purchase_requisition_auto_rfq and
purchase_requisition_bid_selection

gurneyalex committed:
new addon purchase_requisition_auto_rfq_bid_selection

bridge module between purchase_requisition_auto_rfq and
purchase_requisition_bid_selection

gurneyalex commented:
This PR depends on
- [x] #26
- [x] #28

gurneyalex committed:
cleanup: remove useless copied code.

gurneyalex committed:
fix infinite recursion in super() call

yvaucher committed:
[PEP8]

yvaucher commented:
@gurneyalex in tests kitchenset reference is missing

Is a demo data file missing ?

gurneyalex committed:
fix record reference in test

gurneyalex committed:
fix tests

follow api changes introduced in ed88ef0a
use new api in test
fix inverted condition in code

lepistone commented on purchase_requisition_auto_rfq_bid_selection/__openerp__.py:
```diff
@@ -0,0 +1,42 @@
+ 'complexity': "normal",
+ "images": [],
+ "description": """Compatibility module for purchase_requisition_auto_rfq and
+    purchase_requisition_bid_selection.
```
I suggest to add something like:

Specifically, purchase_requisition_auto_rfq allows to create automatically RFQs for a tender, purchase_requisition_bid_selection distinguishes between bids and rfqs. This bridge module takes care that automatically generated documents are either bids of RFQs depending on the mode of the tender.

lepistone commented:
I suggested a new paragraph for the module description. I can add that myself while merging.

Otherwise :+1: thanks.

yvaucher commented:
:+1:

gurneyalex commented:
@lepistone change approved

lepistone committed:
expand doc, move it to README.rst
